Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs

```diff
@@ -23,23 +23,24 @@ public BacktraceDatabaseAttachmentManager(BacktraceDatabaseSettings settings)
 
         public IEnumerable<string> GetReportAttachments(BacktraceData data)
         {
+            var attachmentPrefix = data.Uuid.ToString();
```

Comment:
Is this change for performance (i.e: one ToString call at the top vs. ToString calls in each function)?

Reply:
I added it to avoid unboxing c# type when we want to generate file name. If we will pass guid to string method, it will probably cast it to object and use ToString. To avoid unnecessary allocation we're doing it once before all methoda